### PR TITLE
feat: Allow to save contact from messages list

### PR DESCRIPTION
**Pull request by xavierbriole**
Demo video: https://share.getcloudapp.com/nOuee1Oj

### CODE REVIEW

File: src_htmlPhone/src/components/messages/MessagesList.vue [hunk 1]
```diff
@@ -33,6 +33,7 @@ export default {
       this.disableList = true
       Modal.CreateModal({
         choix: [
+          {id: 7, title: this.IntlString('APP_MESSAGE_SAVE_CONTACT'), icons: 'fa-sms'},
```
Comment (xavierbriole) [inline src_htmlPhone/src/components/messages/MessagesList.vue:36]: Need to change icon

**Comment by Gannon001** [comment]
Why not, is good option.
But need to show this option only if this is not register

**Comment by xavierbriole** [comment]
@Gannon001 oups indeed it’s better ahah

You don’t have to merge it if you don’t want, this is mainly a PR to improve my skill

**Comment by xavierbriole** [comment]
There's another way to check if contact exist or not.

With regex but it may not works in some cases : 
```js
/^[0-9\\-]+$/.test(messGroup[key].display)
```

**Comment by Gannon001** [comment]
Haha, Regex is bad idéa :p

The best method in my opinion: 

https://github.com/N3MTV/gcphone/blob/4be479301f4d376b6901a70b913cfc01b0bbbfb5/src_htmlPhone/src/components/messages/MessagesList.vue#L78

```js
data.unknowContact = contact === undefined
// ...
let mess = []
      Object.keys(messGroup).forEach(key => {
        mess.push({
            // ...
            letter: messGroup[key].letter,
            unknowContact: messGroup[key].unknowContact
```

**Comment by xavierbriole** [comment]
@Gannon001 oh yeah, best way from far ! (the right way lol)

thanks

**Pull request merged**